pkg/odo/cli/version/version.go

```diff
@@ -23,58 +23,100 @@ var (
 	GITCOMMIT = "HEAD"
 )
 
-var clientFlag bool
-
-// versionCmd represents the version command
-var versionCmd = &cobra.Command{
-	Use:   "version",
-	Short: "Print the client version information",
-	Long:  "Print the client version information",
-	Example: `  # Print the client version of Odo
-  odo version
-	`,
-	Run: func(cmd *cobra.Command, args []string) {
-
-		// If verbose mode is enabled, dump all KUBECLT_* env variables
-		// this is usefull for debuging oc plugin integration
-		for _, v := range os.Environ() {
-			if strings.HasPrefix(v, "KUBECTL_") {
-				glog.V(4).Info(v)
-			}
+// RecommendedCommandName is the recommended version command name
+const RecommendedCommandName = "version"
+
+var versionLongDesc = ktemplates.LongDesc("Print the client version information")
+
+var versionExample = ktemplates.Examples(`
+# Print the client version of Odo
+%[1]s`,
+)
+
+// Options encapsulates all options for odo version command
+type Options struct {
```

Comment:
Name should be `VersionOptions` to follow the conventions.

Reply:
That's what I originally gave it.  I changed after seeing golint rec as call would become version.VersionOptions

Reply:
I agree but that's how `oc` does it… we could of course chose not to do it that way but we'd need to discuss it first.

Reply:
Renamed